src/routes/(console)/project-[region]-[project]/overview/platforms/createAndroid.svelte

```diff
@@ -61,6 +61,12 @@ const val APPWRITE_PUBLIC_ENDPOINT = "${sdk.forProject(page.params.region, page.
             trackEvent(Submit.PlatformCreate, {
                 type: 'android'
             });
+
+            addNotification({
+                type: 'success',
+                message: 'Platform created.'
+            });
```

Comment:
don't think these are needed, the platform created section only shows after a successful creation.